### PR TITLE
updating media queries

### LINKED ISSUE #45: updating media queries
hey its jake again im doing more !

### PR DESCRIPTION
**Pull request by JakeDangIt**
idk what else to do so heres more media queries

### CODE REVIEW

File: siths-redesign/components/MissionStatement.vue [hunk 1]
```diff
@@ -51,10 +51,9 @@
                 communication, creativity, and critical thinking skills. We strive to instill in our
                 students a sense of wonder, curiosity, and a growth mindset to meaningfully
                 contribute to and shape a society that values equity, excellence, and empathy. (D31
-                Mission/Vision)
```
Comment (whydoesthecodenotwork) [inline siths-redesign/components/MissionStatement.vue:54]: that's gone amen

**Review by whydoesthecodenotwork — APPROVED** [review]
sure why not

**Review by whydoesthecodenotwork — DISMISSED** [review]
bring `Mission/Vision)` back

File: siths-redesign/components/MissionStatement.vue (hunk 1 above)
Comment (JakeDangIt) [reply siths-redesign/components/MissionStatement.vue:54]: > that's gone amen

idk how it disapeared

**Comment by JakeDangIt** [comment]
> bring `Mission/Vision)` back

its back

**Comment by whydoesthecodenotwork** [comment]
image
carousel overflows when approaching the breakpoint from a larger screen width (lim x->768+)

image
upcoming events gets uncomfortably close to the edge of screen when lim x->768+

File: siths-redesign/components/MissionStatement.vue
Comment (whydoesthecodenotwork) [inline siths-redesign/components/MissionStatement.vue:1]: i like how you dropped this commit then fell asleep in multi

Comment (bridgetf7) [reply siths-redesign/components/MissionStatement.vue:1]: valid

Comment (JakeDangIt) [reply siths-redesign/components/MissionStatement.vue:1]: unable to reproduce, both in devtools and port forwarding on phone and tablet landscape + portrait

Comment (JakeDangIt) [reply siths-redesign/components/MissionStatement.vue:1]: nevermind i see the issue

Comment (JakeDangIt) [reply siths-redesign/components/MissionStatement.vue:1]: completed now

**Comment by whydoesthecodenotwork** [comment]
that's good amen

**Review by JakeDangIt — COMMENTED** [review]
i need someone to review so my changes can be merged

**Comment by bridgetf7** [comment]
> i need someone to review so my changes can be merged

Will do this afternoon-tonight 
Out rn

**Review by bridgetf7 — APPROVED** [review]

**Pull request merged**